frontend/src/features/contacts/contact/detail/Organization.tsx

```diff
@@ -0,0 +1,215 @@
+}
+
+const OrganizationView: React.FunctionComponent<OrganizationViewProps> = ({ organization }) => {
+  const phoneTypes: Dictionary<string> = {};
+  phoneTypes[ContactMethodTypes.WorkMobil] = 'Mobile';
+  phoneTypes[ContactMethodTypes.WorkPhone] = 'Work';
+  phoneTypes[ContactMethodTypes.PersPhone] = 'Home';
+  phoneTypes[ContactMethodTypes.Fax] = 'Fax';
+
+  const personPhoneNumbers: ContactInfoField[] = getContactInfo(organization, phoneTypes);
+
+  const emailTypes: Dictionary<string> = {};
+  emailTypes[ContactMethodTypes.WorkEmail] = 'Work';
+  emailTypes[ContactMethodTypes.PerseEmail] = 'Personal';
+  const personEmails: ContactInfoField[] = getContactInfo(organization, emailTypes);
+
+  let personAddresses: AddressField[];
+  if (organization.addresses === undefined) {
+    personAddresses = [];
+  } else {
+    personAddresses = organization.addresses.reduce(
+      (accumulator: AddressField[], value: IContactAddress) => {
+        accumulator.push({ label: value.addressType.description, address: value });
+
+        return accumulator;
+      },
+      [],
+    );
+  }
+
+  return (
+    <>
+      <FormSection key={'contact-org-' + organization.id + '-names'} className="mb-4">
+        <Styled.RowAligned>
+          <Col>
+            <Styled.H2>
+              <FaRegBuilding size={20} className="mr-2" />
+              <span data-testid="contact-organization-fullname">{organization.name}</span>
+            </Styled.H2>
+          </Col>
+          <Col md="auto" className="ml-auto">
+            <Styled.StatusIndicators className={organization.isDisabled ? 'inactive' : 'active'}>
+              <FaCircle size={7} className="mr-2" />
+              <span data-testid="contact-organization-status">
+                {organization.isDisabled ? 'INACTIVE' : 'ACTIVE'}
+              </span>
+            </Styled.StatusIndicators>
+          </Col>
+        </Styled.RowAligned>
+        <Styled.RowAligned>
+          <Col className="ml-5">
+            <div>
+              <strong>Alias:</strong>
+            </div>
+            <div>
+              <span data-testid="contact-organization-alias">{organization.alias}</span>
+            </div>
+            <div>
+              <strong>Incorportation Number:</strong>
+            </div>
+            <div>
+              <span data-testid="contact-organization-incorporationNumber">
+                {organization.incorporationNumber}
+              </span>
+            </div>
+          </Col>
+        </Styled.RowAligned>
+      </FormSection>
+      <FormSection key={'contact-org-' + organization.id + '-contacts'} className="mb-4">
+        <Styled.H2Primary>Contact info</Styled.H2Primary>
+        <Styled.RowAligned className="pb-4">
+          <Col md={2}>
+            <strong>Email:</strong>
+          </Col>
+          <Col>
+            {personEmails.length === 0 && <span>N.A</span>}
+            {personEmails.map((field: ContactInfoField) => (
+              <Styled.RowAligned>
```

Comment:
react will probably complain that you are missing a key here. (keys are required when using map to generate jsx from an array).

Reply:
same comment applies to other usages of map (I don't want to clog this review with duplicate comments).